content/applications/hr/payroll.rst

```diff
@@ -526,34 +696,35 @@ Resume
 
 .. note::
    Currently, the :guilabel:`Resume` feature found inside the :menuselection:`Payroll app -->
-   Configuration --> Salary Package Configurator --> Resume` menu is still in development and only
+   Configuration --> Salary Package Configurator: Resume` menu is still in development and only
    serves a specific use case for Belgian markets. The documentation will be updated when this
    section has matured to more markets.
 
-Offers
-------
+Jobs
+====
+
+Since the *Payroll* application is responsible for paying employees for specific job positions, the
+complete list of job positions can be found in both the *Payroll* and *Recruitment* applications.
 
-When a candidate is offered a position, there are several items that need to be tracked in order
-for a business to stay organized, such as where in the offer process the candidate is, how long the
-offer is valid for, as well as all the offer details. These details are all stored in each *offers*
-record.
+.. _payroll/job-positions:
 
-To view all offers, go to :menuselection:`Payroll --> Configuration --> Salary Package Configurator
---> Offers`. All offers that have been sent to either potential candidates or current employees
-appears in this list. The status, offer start date and expiration date, amount of the contract, and
-more, can all be found in this list. Offers sent via the :guilabel:`Recruitment` application appear
-here, but there is an option to create a new offer from the :guilabel:`Payroll` application as well.
+Job positions
+-------------
 
-.. image:: payroll/offers.png
+The job positions listed in the *Payroll* application are identical to the job positions listed in
+the *Recruitment* application. If a new job position is added in the *Recruitment* application, it
```

Comment:
I feel like the first 3 sentences in this section are essentially saying the same thing. Are they all necessary?

Reply:
They are but they are slightly different- so I do think I'll keep them.